OneCommunityGlobal/HighestGoodNetworkApp
# Aaron remove infinite scroller

# Description
- removing infinite scroller for displaying Team Member Tasks on the dashboard and going back to the datatable
- also removed a lot of tasks for users who had 100+ tasks

## Related PRS (if any):
- reverting the infinite scroller changes made in [PR1059](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1059)

## Main changes explained:
- undoing the changes made in PR1059 (removing infinite scroll component)
- resolved most tasks for "Haohui Lin" and "Shivansh Test4" on dev branch so only 4-5 remain incomplete

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log in as owner/admin
 3a. go to dashboard
 3b. make sure the "Team Member Tasks" loads in a reasonable amount of time and doesn't freeze
4. log in as volunteer/manager/mentor
 4a. go to dashboard
 4b. make sure the "Team Member Tasks" loads in a reasonable amount of time and the page doesn't crash with a white screen

## Screenshots or videos of changes:

<b>Loading Dashboard</b>

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/10265513/6c155af1-d86c-4248-9412-5c64ba53818d

<b>Users with 100+ tasks</b>

![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/10265513/37f909eb-2ab9-4c90-968a-14394f6713ca)

![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/10265513/742df15f-9e48-4e8f-a554-18d93d41fdde)

- resolved almost all the tasks from these users, leaving 4-5 tasks
